projects/js-toolkit/docs/manuals/liferay-cli.md

```diff
@@ -53,6 +57,52 @@ Once the project is created, you may change to its folder and run
 `npm|yarn install` to install the dependencies, then invoke the supported
 actions (like `build`, `deploy`, etc.) with `npm|yarn run <command>`.
 
+#### Project types
+
+Currently the following project types are supported.
+
+##### Liferay Platform Project
+
+The typical project from JS Toolkit that creates a deployable JAR file
+implementing a Widget (portlet) that can be added to any page.
+
+The project may support different features like
+[localization](./features/localization.md) and
+[configuration](./features/configuration.md), and it follows the
+[JavaScript portlet entry point](./reference/js-portlet-entry-point.md)
+contract.
+
+##### Liferay Remote App
+
+Remote Apps implement a
+[Custom Element](https://developer.mozilla.org/en-US/docs/Web/Web_Components/Using_custom_elements)
+that can be registered with Liferay DXP in the Remote Apps configuration. This
+generates a new Widget that is rendered using the custom element, instead of
+the old [JavaScript portlet entry point](./reference/js-portlet-entry-point.md)
```

Comment:
sometimes people think old === bad, so probably best to leave that out since it is still a primary mechanism for js portlets

```suggestion
using the [JavaScript portlet entry point](./reference/js-portlet-entry-point.md)
```